.npmrc

```diff
@@ -1 +0,0 @@
-registry=https://registry.npmjs.org/
```

Comment:
This can be useful so we don't accidentally install things using GoDaddy's private registry (which would affect the lockfile).